src/test/java/core/basesyntax/service/csvfileservice/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service.csvfileservice;
```

Comment:
```suggestion
package core.basesyntax.service.file.scv;
```
scvfilereader is a weird name for package